Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rustlang/rust:nightly
+
+WORKDIR /meilisearch
+
+COPY . .
+
+RUN cargo build
+
+CMD cargo run
```

Comment:
Wouldn't it be preferable to compile using `cargo build --release`, copy the `/target/release/meilidb-http` binary into _another_ sub image. This way we avoid compiling it every times it run and avoid keeping useless dependencies like `cargo`, `gcc`.

Reply:
yup ;)